packages/rocketchat-ui/client/views/app/room.js

```diff
@@ -856,6 +856,18 @@ Template.room.events({
 			Meteor.call('sendMessage', msgObject);
 		});
 	},
+	'click .js-actionButton-respondWithMessage'(event) {
+		const msg = event.currentTarget.value;
+		if (!msg) {
+			return;
+		}
+
+		const input = $('.messages-container .rc-message-box .rc-message-box__textarea').get(0);
```

Comment:
this should use `chatMessages` object, like [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/ui-message/client/startup/restoreText.js#L18)